packages/main/src/themes/Toast.css

```diff
@@ -0,0 +1,84 @@
+.ui5-toast-root-parent {
+	position: fixed;
+	left: 0;
+	right: 0;
+	top: 0;
+	bottom: 0;
+	width: 100%;
+	display: none;
+	height: 100%;
+	pointer-events: none;
+	overflow: hidden;
+	z-index: 999;
+	justify-content: center;
+	align-items: flex-end;
+}
+
+.ui5-toast-text {
+	font-family: var(--sapUiFontFamily);
+	color: var(--sapUiListTextColor);
+	font-weight: normal;
+	font-size: var(--sapMFontMediumSize);
+	word-wrap: break-word;
+	text-align: center;
+	text-decoration: none;
+	text-overflow: ellipsis;
+	white-space: pre-line;
+}
+
+.ui5-toast-root {
```

Comment:
minor one (just if you think it is better), move `.ui5-toast-root` before the `.ui5-toast-text` to order them as they are in DOM hierarchy.